### ISSUE TITLE
Make pull request comment better for monorepo

### ISSUE THREAD

**Issue by dragon3** [k1LoW/octocov#152]
We use the `octocov` in our monorepo which has multiple language directories, like this:

```
repo
├── .octocov.go.yaml
├── .octocov.java.yaml
├── .octocov.python.yaml
├── go
├── java
└── python
```
We test, generate coverage file, and run `octocov` individually with the separated config files and GitHub actions workflows.

Sometimes we need to run those workflows at the same time (e.g. build whole repository), then comments on pull request look like this:

<img width="941" alt="Screen_Shot_2022-04-14_at_15_37_22" src="https://user-images.githubusercontent.com/20374/163363897-95e7873f-61d0-4584-a151-4feea8edb34c.png">

As you can see, there are 3 different reports, but it's hard to know which report is for which.
Also comments got hidden (minimized) except for the last one even they are different reports.

---

So it would be great if:
- each report can have comment header (title), like
    - `Code Metrics Report (go)`
    - `Code Metrics Report (java)`
    - `Code Metrics Report (python)`
- they will not hidden by other reports.

What do you think?

**Comment by k1LoW** [comment]
Thank you for the nice suggestion!

I'll try to implement it.

**Comment by k1LoW** [comment]
Released v0.40.0 fixing this problem.
It determines differences in reports in the repository: section of .octocov.yml.

**Comment by dragon3** [comment]
Thank you for the quick work!

I tried the version `v0.40.0` and confirmed it works perfectly 😄 👍 

<img width="843" alt="screenshot" src="https://user-images.githubusercontent.com/20374/163700406-b921329d-f9fb-431e-9c95-67988d37a55d.png">
